spec/lib/traffic_source_parser/parser/referrer_parser_spec.rb

```diff
@@ -0,0 +1,217 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.